### PR TITLE
Add start on #29

### PR DESCRIPTION
**Pull request by robinlovelace-ate**

### CODE REVIEW

**Comment by robinlovelace-ate** [comment]
Please review @becfleming-ate.

File: github.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+The process is as follows:
+
+1. Create an issue in GitHub
+  - For example, this document was created in support of https://github.com/acteng/overview/issues/29
+2. Create a branch for the issue
+  - With the gh command line tool you can do this as follows:
+    - `gh issue develop 29 --checkout`
+    - You should now be on the new branch, the command above for example created this branch: https://github.com/acteng/overview/tree/29-documentation-on-how-we-use-github
+    - Do your work in the branch, adding commits regularly and with clear commit messages to document your work
```
Comment (becfleming-ate) [inline github.md:14]: I think there is some missing @robinlovelace-ate

Comment (robinlovelace-ate) [reply github.md:14]: Ah yes, good spot. Will add now.

**Review by becfleming-ate — APPROVED** [review]
Looks good to me Robin

File: github.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Using GitHub in Active Travel England
+
+This is a guide to using GitHub in Active Travel England.
+We use a process called Git Flow to manage our work.
+
+The process is as follows:
+
+1. Create an issue in GitHub
+  - For example, this document was created in support of https://github.com/acteng/overview/issues/29
```
Comment (becfleming-ate) [inline github.md:9]: could usefully add the command for creating an issue in here:

gh issue create

Comment (robinlovelace-ate) [reply github.md:9]: Agreed. Can you add that please Bec?

Comment (robinlovelace-ate) [reply github.md:9]: Will merge in any case, can add after.

**Pull request merged**